component/core/src/main/java/org/exoplatform/social/core/service/LinkProvider.java

```diff
@@ -471,7 +471,6 @@ private static String buildAttachmentUrl(String providerId,
     }
 
     if (lastModifiedDate == null || lastModifiedDate <= 0 || lastModifiedDate == DEFAULT_IMAGES_LAST_MODIFED) {
-      id = DEFAULT_IMAGE_REMOTE_ID;
```

Comment:
We must keep this to preserve the current behavior concerning default avatars for other type of identities